pkg/resource/logging-credentials/reconciler.go

```diff
@@ -0,0 +1,79 @@
+package loggingcredentials
+
+import (
+	"context"
+	"fmt"
+
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// Reconciler implements a reconciler.Interface to handle
+// Logging Credentials: store and maintain logging credentials
+type Reconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+// ReconcileCreate ensures a secret exists for the given cluster.
+func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Interface) (ctrl.Result, error) {
+	logger := log.FromContext(ctx)
+
+	const secretNamespace = "monitoring"
+	const secretName = "logging-credentials"
```

Comment:
Usually its good to have those in a place where you can re-use them, as we later will need to refer to this object. That's why we usually have them in a ObjectMeta which is exposed vi an exported function.

Reply:
Well, my latest commit created a common loggingConfig where we can put various config stuff, but maybe you have better ideas?